pom.xml

```diff
@@ -425,6 +425,11 @@
     </pluginRepositories>
     
     <dependencies>
+     <dependency>
+        <groupId>org.apache.httpcomponents.core5</groupId>
+        <artifactId>httpcore5-testing</artifactId>
+        <version>5.1</version>
+      </dependency>
```

Comment:
Since this is a testing dependency, please move it to where the other test dependencies are and add the "test" scope to it. Also, please have it derive the version number from the appropriate POM property.

Reply:
updated